packages/react/src/ButtonGroup/ButtonGroup.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import {ComponentMeta, ComponentStory} from '@storybook/react'
+import React from 'react'
+import {ButtonGroup} from '.'
+import {Button} from '../Button'
+
+export default {
+  title: 'Components/ButtonGroup',
+  component: ButtonGroup
+} as ComponentMeta<typeof ButtonGroup>
+
+const Template: ComponentStory<typeof ButtonGroup> = args => (
+  <ButtonGroup {...args}>
+    <Button>This is one button</Button>
+    <Button>This is the second button</Button>
+  </ButtonGroup>
+)
+
+export const Primary = Template.bind({})
```

Comment:
Forgot to mention earlier, but the first story will be called 'Playground' going forward, and all controls should be configurable...

Related ADR: https://github.com/github/primer/pull/1387

While PR is still open, it's still very likely to be merged soon so good to pre-emptively align with it.

Reply:
This is updated.